README.md

```diff
@@ -85,8 +85,8 @@ Only tested code will be accepted. Please follow fix the style guide.
 
```

Comment:
Oh - can confirm that it's working despite my scepticism, I put in your `$options` snippet above and it changed the indentation of the arrow slightly. :D